setup.py

```diff
@@ -60,7 +62,7 @@
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
     ],
-    test_suite='tests',
+    test_suite='nose.collector',
```

Comment:
Could you talk a bit about why we're moving to `nose`?

Reply:
There was a small note in the commit.

Basically `python setup.py test` was swallowing some simple errors (such as that the requests package wasn't installed.

`nose` presented these errors clearly and reminded me of my obvious error.